src/main/java/com/glisco/things/items/trinkets/AgglomerationItem.java

```diff
@@ -53,6 +62,80 @@ public AgglomerationItem() {
         super(new Item.Settings().maxCount(1).rarity(Rarity.UNCOMMON));
     }
 
+    //--------
+
+    @Override
+    public boolean onClicked(ItemStack stack, ItemStack otherStack, Slot slot, ClickType clickType, PlayerEntity player, StackReference cursorStackReference) {
```

Comment:
Generally this implementation is fine, not a large fan of `stack1` though - giving it a more descriptive name like `innerStack` or something similar is preferrable